contracts/Escrow.sol

```diff
@@ -38,14 +39,14 @@ contract Escrow is ChainlinkClient, Initializable, ContextUpgradeable, OwnableUp
     address private oracle;
     bytes32 private jobId;
 
-    enum OfferStatus { NON, OFFER_CREATED, OFFER_COMPLETED, OFFER_CANCELLED, OFFER_WITHDRAWN }
-    enum UserStatus  { NON, OPEN, DEPOSIT, CLAIM }
+    enum OfferStatus { NON, OFFER_CREATED, OFFER_ACCEPTED, OFFER_ACTIVE, OFFER_COMPLETED, OFFER_FINALIZED, OFFER_TIMEDOUT, OFFER_CANCELLED }
 
+    // Do we need this? TODO: Remove if not.
     struct OfferCounterpart {
         bytes32 commitment;
         uint256 amount;
         address partyAddress;
-        UserStatus partyStatus;
+        bool cancel;
     }
 
     struct Deal {
```

Comment:
Is there a good reason to keep the OfferId in the struct of the Deal? Isn't it implied that you need the Id to find the Deal?

Reply:
oh lul removed.